lib/senkyoshi/canvas_course.rb

```diff
@@ -153,8 +162,8 @@ def upload_scorm_package(scorm_package, course_id, tmp_name)
     # Creates assignments from all previously uploaded scorm packages
     ##
     def create_scorm_assignments(scorm_packages, course_id, local)
-      scorm_packages.each do |pack|
-        create_scorm_assignment(pack, course_id, local)
+      scorm_packages.map do |pack|
```

Comment:
As long as the method is explicit about its behavior, I don't think it's a problem for it to return the assignments it creates. Just make it clear (in a comment, for instance) that `#create_scorm_assignment`) returns the assignment it creates, and `#create_scorm_assignments` returns an array of all created assignments.

The issue with the code as it is now (to me, anyway) is that the behavior feels too implicit, and so writing a test that depends on the return value of the method feels like you're depending on internal behavior.